src/overview/actions.js

```diff
@@ -56,6 +56,9 @@ export const toggleDomainFilter = createAction('overview/toggleDomainFilter')
 export const setTagFilters = createAction('overview/setTagFilters')
 export const setDomainFilters = createAction('overview/setDomainFilters')
 
+export const showTooltip = createAction('overview/showTooltip')
+export const tooltipIndex = createAction('overview/tooltipIndex')
```

Comment:
Maybe this action isn't need anymore, but just remember to always start your action names with one of the conventional verbs like `set`, `toggle`, `inc`, etc., depending on what it does. It becomes a lot easier to understand how the state changes.